toolchain-fitnesse-plugin/src/test/java/nl/praegus/fitnesse/responders/testHistory/RecentTestHistoryResponderTest.java

```diff
@@ -62,6 +63,24 @@ public void Checks_number_of_times_passed(){
 
         assertThat(recentTestHistory.getHistoryLines().get(0).getNumberOfTimesPassed()).isEqualTo(0);
     }
+    @Test
+    public void Checks_if_filter_is_off(){
+        RecentTestHistory recentTestHistory = new RecentTestHistory(getMockDir("TestResultDirectory"));
+        List<String> expectedResult = Arrays.asList("Example.Mocker3.SuiteSetUp","Example.Mocker3.SuiteTearDown","Example.Mocker3.SetUp","Example.Mocker3.TearDown");
+
+        List receivedResult = recentTestHistory.getHistoryLines();
+
+        assertThat(receivedResult).extracting("pageName").containsAnyElementsOf(expectedResult);
+    }
+    @Test
+    public void Checks_if_filter_is_on(){
+        RecentTestHistory recentTestHistory = new RecentTestHistory(getMockDir("TestResultDirectory"));
+        List<String> expectedResult = Arrays.asList("Example.Mocker3.SuiteSetUp","Example.Mocker3.SuiteTearDown","Example.Mocker3.SetUp","Example.Mocker3.TearDown");
+
+        List receivedResult = recentTestHistory.getFilteredTestHistoryLines();
+
+        assertThat(receivedResult).extracting("pageName").doesNotContainAnyElementsOf(expectedResult);
```

Comment:
Hier wordt het helemaal verwarrend. Er staat nu dat je niet verwacht dat er verwachte resultaten zijn...

Reply:
dit is omdat als het filter aan staat ik niet de tests terug wil waar "setup" en "teardown" in de naam zit, hier controlleer ik dan op.